kubernetes-deploy.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "googleauth", "~> 0.6.6" # https://github.com/google/google-auth-library-ruby/issues/153
   spec.add_dependency "ejson", "~> 1.0"
   spec.add_dependency "colorize", "~> 0.8"
-  spec.add_dependency "statsd-instrument", "~> 2.3"
+  spec.add_dependency "statsd-instrument", ">= 2.3.2"
```

Comment:
This will automatically install a version `3.x` gem if it becomes available (or even `2.4.x`), no? `~> 2.3.2` might be safer in that respect.

Reply:
Yes, you're right. I think what we really want is `'~> 2.3', '>= 2.3.2'` (i.e. same flexibility as before on minor versions, but mininum 2.3.2)